browser/brave_vpn/win/wireguard_utils_win.cc

```diff
@@ -111,7 +181,22 @@ void EnableBraveVpnWireguardService(const std::string& server_public_key,
                                     const std::string& client_private_key,
                                     const std::string& mapped_ip4_address,
                                     const std::string& vpn_server_hostname,
+                                    std::optional<std::string> smart_proxy_url,
                                     wireguard::BooleanCallback callback) {
+  // If all params are empty this is a reconnect (using last known good config).
+  // This happens when the person triggered reconnect using the tray icon.
+  bool reconnect_using_last_config =
+      server_public_key.length() == 0 && client_private_key.length() == 0 &&
+      mapped_ip4_address.length() == 0 && vpn_server_hostname.length() == 0;
```

Comment:
nit: your choice if you want to change or not, but `server_public_key.empty()` is slightly shorter than `server_public_key.length() == 0`.